tensorflow/lite/micro/testing/micro_test.h

```diff
@@ -63,8 +63,21 @@ extern int tests_failed;
 extern bool is_test_complete;
```

Comment:
I'd like to avoid having target-specific logic in the testing library. No suggestions at this time, will try and think about it tomorrow.

Reply:
Let's take an approach similar to debug_log, to add a target-specific hook for initialization.

micro/system_setup.h:
```cc
void Initialize();
```
 
micro/system_setup.cc:
```cc
void Initialize() {}
```

micro/cortex_m_cornstone_300/system_setup.cc:
```cc
void Initialize() {uart_init();}
```

And then micro_test.h (and for that matter any other common binary, like the benchmarks) can add a call to `Initialize()`.

What do you think?

Reply:
Sounds good!